protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx

```diff
@@ -49,8 +54,30 @@ function AnalyticsStateDistribution({ geography, selectedGeographicFeature }) {
   }
 
   let conditionalCaptionJSX;
+  let communityCharacteristics;
 
   if (selectedGeographicFeature) {
+    communityCharacteristics = (
```

Comment:
Something else to consider would be to move the use of the button/link to MainChart instead of AnalyticsStateDistribution as it is sorta independent from the histogram/table.

Reply:
Great idea! Check the second pass to see if I'm on the right track. Can extract the button to main chart next